src/Raven.Server/Documents/Handlers/Processors/Documents/AbstractDocumentHandlerProcessorForGet.cs

```diff
@@ -493,4 +503,228 @@ protected sealed class StartsWithParams
 
         public string StartAfterId { get; set; }
     }
+
+    public sealed class QueryStringParameters : AbstractQueryStringParameters
```

Comment:
Can we move to a dedicated file (`DocumentHandlerQueryStringParameters.cs`) or a partial class?